packages/parser/src/syntax/code.ts

```diff
@@ -6,7 +6,7 @@ export const parseCode = () => {
 
 export const codeMap = [
   {
-    suffix: ['adv', 'advscript', 'json'],
```

Comment:
Why remove `json`?

Reply:
Please check your code. You have set it in pre code.